### ISSUE TITLE
Example for Name / Surname

### ISSUE THREAD

**Issue by michschl** [swissfintechinnovations/common-API#59]
To avoid misunderstanding, we should swap the information max muster

**Comment by simonbbaumgartner** [comment]
Yes, please. This just happened to me while mapping our TKB-fields on the Common API fields. Didn't read thoroughly and just fell into the trap.

**Comment by simonbbaumgartner** [comment]
This is relevant for applicantDetail and tooAdvisorDetail.
